app/code/community/Bolt/Boltpay/Model/Observer.php

```diff
@@ -78,54 +78,43 @@ private function getBoltApiHelper()
      *
      * @param $observer
      * @throws Exception
+     *
      */
     public function verifyOrderContents($observer) 
     {
         $boltHelper = $this->getBoltApiHelper();
+        /* @var Mage_Sales_Model_Quote $quote */
         $quote = $observer->getEvent()->getQuote();
+        /* @var Mage_Sales_Model_Order $order */
         $order = $observer->getEvent()->getOrder();
+        $transaction = $observer->getEvent()->getTransaction();
 
         $payment = $quote->getPayment();
-        $items = Mage::getSingleton('checkout/session')->getQuote()->getAllVisibleItems();
         $method = $payment->getMethod();
 
         if (strtolower($method) == Bolt_Boltpay_Model_Payment::METHOD_CODE) {
-            if (Mage::getStoreConfig('payment/boltpay/auto_capture') == Bolt_Boltpay_Block_Checkout_Boltpay::AUTO_CAPTURE_ENABLED) {
-                $authCapture = true;
-            } else {
-                $authCapture = false;
-            }
 
             $reference = $payment->getAdditionalInformation('bolt_reference');
-            $cart_request = $boltHelper->buildCart($quote, $items, false);
-            $complete_authorize_request = array(
-                'cart' => $cart_request,
-                'reference' => $reference,
-                'auto_capture' => $authCapture
-            );
-
-            try {
-                if (!Mage::getStoreConfig('payment/boltpay/disable_complete_authorize'))  {
-                    $this->sendCompleteAuthorizeRequest($complete_authorize_request);
-                }
-            } catch (Exception $e) {
+
+            if ( (int)($order->getGrandTotal()*100) !== $transaction->amount->amount)  {
+
                 $message = "THERE IS A MISMATCH IN THE ORDER PAID AND ORDER RECORDED.<br>PLEASE COMPARE THE ORDER DETAILS WITH THAT RECORD IN YOUR BOLT MERCHANT ACCOUNT AT: ";
```

Comment:
Let's include `$order->getGrandTotal()*100` and `$transaction->amount->amount` in message.